cfme/tests/cloud/test_tenant.py

```diff
@@ -34,14 +37,21 @@ def test_tenant_crud(tenant):
     Metadata:
         test_flag: tenant
     """
+
     tenant.create(cancel=True)
     assert not tenant.exists()
 
     tenant.create()
     assert tenant.exists()
 
+    tenant.update({'name': 'my_new_name'})
+    refresh_provider(tenant.provider)
+    assert tenant.exists()
```

Comment:
**REQUIRED:** One could argue that you do not need the refresh here as the exists should call it. In actual fact the fact that you don't call the refresh in the update method before starting the wait_for function is confusing as to why you need it later on here. If we didn't need a refresh to make sure that the provider existed in the loop, why do we need it now?